### DISCUSSION TITLE
Provide ability to control HID-Devices

### DISCUSSION THREAD

**Discussion by rofl-er** [MobiFlight/MobiFlight-Connector#384] [Ideas]
As a user of Leo Bodnar boards, it would be awesome to configure them also through Mobiflight.

**Comment by pizman** [comment]
Hi Guys.
I´m not sure if you still got the same idea like me, or if i got a fundametal new inspiration.......
Could this feature be the key for the long time "Potentiometer" thematic ??

If we be able to READ a HID Device like a Bodnar to detect the status of the 32 IO Buttons.....   Then we also would be able to read a 16 Bit Value of a Analog Axis on that HID Controller  ( Same way as it is send to the HID Reciever System on the Computer).

For sure.... we would still need a new Posibility in the GUI to handle these Values in a Input Config ( instead to simple 1/0 or Left/Right a.t.m.)   
BUT
We  could stop thinking about a Firmware solution to allow Potis,  We no longer need to think about Comunication Interface Problem with Poti Values over the Mega ( High serial traffic and polling time) .....     So we could fully ignore the INPUT Interface for that and just need to think about the "output" Execute to the Sim.

Save 50%+ of the work !
What do you think ?

**Reply by rofl-er** [reply]
The idea was to use Mobiflight exclusively for all configurations. Currently I have to use SPAD.NEXT for the Leo Bodnars and MF for all Ardunio boards.

**Reply by tigert** [reply]
This would let us map analog inputs to functions that are not available for mapping in the simulator input configuration.

Mappable inputs you can already work with by reading the value from a sim variable and do stuff based on that)

Reading HID devices directly would mean a whole new interface for the data input though.

**Comment by MobiFlight-Admin** [comment]
This is now available

**Comment by DocMoebiuz** [comment]
We can now read HID devices

**Comment by DocMoebiuz** [comment]
Thanks for this great idea @rofl-er
